app/assets/javascripts/admin/addon/components/emoji-value-list.js

```diff
@@ -0,0 +1,208 @@
+import Component from "@ember/component";
+import I18n from "I18n";
+import { on } from "discourse-common/utils/decorators";
+import { emojiUrlFor } from "discourse/lib/text";
+import { action, set, setProperties } from "@ember/object";
+import { later, schedule } from "@ember/runloop";
+
+export default Component.extend({
+  classNameBindings: [":value-list"],
```

Comment:
what is the reason of this classname?

Shouldn't we have a emoji-list classname too ?

Reply:
indeed thats true